src/main/java/duke/exception/DukeException.java

```diff
@@ -0,0 +1,21 @@
+package duke.exception;
+
+/**
+ * Represents a generic exception in the Duke application.
+ */
+
+public class DukeException extends Exception {
+    public DukeException() {
+        super();
+    }
+
+    /**
+     * Gets a default error message for the exception.
+     *
+     * @return A string representing the default error message.
+     */
+    @Override
+    public String getMessage() {
+        return "Unfortunately ";
```

Comment:
It feels like you do not want a DukeException to be instantiated directly, because it's `getMessage` method would always just have the message `Unfortunately`. Maybe it would be a good idea to make it an abstract class, even if it has no abstract methods?